Source/bmalloc/bmalloc/TZoneHeapManager.h

```diff
@@ -175,6 +193,9 @@ class TZoneHeapManager {
     TZoneHeapManager::State m_state;
     Mutex m_mutex;
     TZoneHeapRandomizeKey m_tzoneKey;
+    unsigned bucketsForSmallSizes { defaultBucketsForSmallSizes };
+    unsigned bucketsForLargeSizes { defaultBucketsForLargeSizes };
+    unsigned maxSmallSize = { defaultMaxSmallSize };
```

Comment:
We can just do `unsigned maxSmallSize { defaultMaxSmallSize };`.

Reply:
Yep.